cli/compose/template/template_test.go

```diff
@@ -81,3 +89,64 @@ func TestNonAlphanumericDefault(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "ok /non:-alphanumeric", result)
 }
+
+func TestMandatoryVariableErrors(t *testing.T) {
+	testCases := []struct {
+		template      string
+		expectedError string
+	}{
+		{
+			template:      "not ok ${UNSET_VAR:?Mandatory Variable Unset}",
+			expectedError: "Mandatory Variable Unset",
```

Comment:
I don't understand this test case. Why would anyone put a default if the value is required?

I don't think the error message should contain the unused default string, it should say something like "required variable UNSET_VAR is missing a value"

Reply:
For reference; this is what it looks like for `/bin/sh` and `Bash`:

```bash
echo ${UNSET_VAR:?Mandatory Variable Unset}
sh: UNSET_VAR: Mandatory Variable Unset

echo ${UNSET_VAR:?Mandatory Variable Unset}
bash: UNSET_VAR: Mandatory Variable Unset
```